packages/usa-file-input/src/index.js

```diff
@@ -28,10 +28,22 @@ const EXCEL_PREVIEW_CLASS = `${GENERIC_PREVIEW_CLASS_NAME}--excel`;
 const SR_ONLY_CLASS = `${PREFIX}-sr-only`;
 const SPACER_GIF =
   "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7";
-
 let TYPE_IS_VALID = Boolean(true); // logic gate for change listener
-let DEFAULT_ARIA_LABEL_TEXT = "";
-let DEFAULT_FILE_STATUS_TEXT = "";
+
+const ERROR_TEXT = "This is not a valid file type.";
+let ARIA_LABEL_TEXT = "";
+let DRAG_TEXT = "";
+let CHANGE_FILE_TEXT = "";
+let FILE_STATUS_TEXT = "";
+let CHANGE_FILE_TEXT_SINGULAR = "Change file";
+let CHANGE_FILE_TEXT_PLURAL = "Change files";
+let CHOOSE_TEXT = "choose from folder";
+let DRAG_TEXT_SINGULAR = "Drag file here or";
+let DRAG_TEXT_PLURAL = "Drag files here or";
+let NO_FILE_TEXT_SINGULAR = "No file selected.";
+let NO_FILE_TEXT_PLURAL = "No files selected.";
+let SELECTED_FILE_TEXT_SINGULAR = "file selected";
+let SELECTED_FILE_TEXT_PLURAL = "files selected";
```

Comment:
## Optional

Should we replace `PLURAL` with `MULTIPLE` to follow existing conventions for these?